src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,13 @@
+$main-font-color: rgb(6, 11, 53);
+$secondary-font-color: rgb(97, 96, 112);
+
+$blue-color: #00acdc;
+
+$cardNumber: 8;
+$fontSize: 12px;
```

Comment:
use `kebab-case` syntax for all css variables:
```
$card-number
$font-size
```

Reply:
changed